packages/vuetify/src/components/VDataTable/VDataTableHeaders.tsx

```diff
@@ -102,6 +110,9 @@ export const VDataTableHeaders = genericComponent<VDataTableHeadersSlots>()({
 
     const { backgroundColorClasses, backgroundColorStyles } = useBackgroundColor(props, 'color')
 
+    const { mobile } = useDisplay()
+    const mobileView = computed(() => typeof props.mobile !== 'undefined' ? props.mobile : mobile.value)
```

Comment:
this could ultimately be handled by useDisplay if passed props

Reply:
Pushed a change for this, let me know if I did this correctly and/or what you were looking for.

Reply:
Pushed a change for this, let me know if I did this correctly and/or what you were looking for.